config/config.go

```diff
@@ -22,12 +22,12 @@ package config
 
 import (
 	"context"
+	"fmt"
 	awssdk "github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/elastic/beats/v7/libbeat/processors"
 	"github.com/elastic/beats/v7/x-pack/libbeat/common/aws"
 	"github.com/elastic/elastic-agent-libs/config"
 	"github.com/elastic/elastic-agent-libs/logp"
-	"gopkg.in/yaml.v3"
 	"time"
 )
```

Comment:
Not really a part of your PR, but do you mind grouping these imports into three groups separated by empty lines: standard library imports, Elastic org imports, third-party imports.
```
import (
	"context"
	"fmt"
	"time"
	
	"github.com/elastic/beats/v7/libbeat/processors"
	"github.com/elastic/beats/v7/x-pack/libbeat/common/aws"
	"github.com/elastic/elastic-agent-libs/config"
	"github.com/elastic/elastic-agent-libs/logp"
	
	awssdk "github.com/aws/aws-sdk-go-v2/aws"
)
```

Reply:
Sure thing